app/src/main/dist/config/crate.yml

```diff
@@ -34,9 +34,8 @@
 #gateway.recover_after_nodes: 2
 #discovery.zen.minimum_master_nodes: 2
 
-# most cloud environments don't support multicast discovery
+# CrateDB doesn't support multicast discovery
```

Comment:
I think the line should be removed completely. Without the context that it once did support multicast this doesn't make sense